### ISSUE TITLE
The energy consumption resource properties in power-uart.js don't comply with the OCF spec

### ISSUE THREAD

**Issue by qiuzhong** [01org/SmartHome-Demo#63]
According to the OCF spec, the energy consumption resource type should contain two properties: [power](https://github.com/01org/SmartHome-Demo/blob/master/ocf-servers/doc/oic.r.energy.consumption.json#L9) and [energy](https://github.com/01org/SmartHome-Demo/blob/master/ocf-servers/doc/oic.r.energy.consumption.json#L13)

However, in the current **power-uart.js** implmentation, the two properties are named as: [power1](https://github.com/01org/SmartHome-Demo/blob/master/ocf-servers/js-servers/power-uart.js#L107) and [power2](https://github.com/01org/SmartHome-Demo/blob/master/ocf-servers/js-servers/power-uart.js#L108)

**Comment by ttzeng** [comment]
I've raised same concern in the issue #29 :)

**Comment by kiendinh** [comment]
It's the custom one and thus use the custom definitions.

**Comment by gvancuts** [comment]
I could see how we could convert one of the channel into a energyConsumption property (where the "device" is the entire house). But the definition given to the energyConsumption property (see below) does not apply to the Solar Panel electricity production. Maybe the right course of action here would be to transform channel 1 into an energyConsumption model and keep the Solar Panel properties custom?

[1] http://oneiota.org/revisions/1761
/EnergyConsumptionResURI:
  description: |
    This resource describes the energy consumed by the device since power up (the energy value is in Watt Hours [Wh]) 
    and the instantaneous power draw of the device (the power value is in Watts [W]) at the time the resource was queried.
    The power value is in Watts [W].
    The energy value is in Watt Hours [Wh].

**Comment by ttzeng** [comment]
Why not follow the "Vendor specified extensions" specifiation called out in the section 5.5 of the [OCF Resource Type Specification](https://openconnectivity.org/wp-content/uploads/2016/06/OIC_1.1_Candidate_Specification.zip), and use 'x.com.intel.energy.consumption' as the 'rt' property?

**Comment by qiuzhong** [comment]
Hi, @gvancuts, I really appreciate your clarification.

I agree that
> the definition given to the energyConsumption property (see below) does not apply to the Solar Panel electricity production.

However, from an OCF client's prospective, all the energy comsumption resources are found via the type name <code>oic.r.energy.consumption</code> and interface name <code>/a/power</code>. Currently, the DC Meter is using the Energy Consumption resource type name (and resource interface name) defined in OCF spec though they have different meanings for the properties. So the OCF client can't recognize the custom properties. 

In the smart-home-control OCF client, I tried presenting the energy comsumption resource. According to the [doc](https://github.com/01org/SmartHome-Demo/blob/master/ocf-servers/doc/oic.r.energy.consumption.json), I refered the power and energy properties. But I found they're undefined after the sensor data object is recevied in the client. There is no documentation about the custom properties(power1 and power2). I had to figure out them in power-uart.js.

Either keeping align with OCF spec or introducing new resource type for DC Meter particularly will be fine for the OCF client.

In terms of your suggestion
> transform channel 1 into an energyConsumption model and keep the Solar Panel properties custom

So you suggest to hold the properties defined in OCF spec for the energy comsumption resource and append the custom Solar panel properties to it?

What about the suggestion from @ttzeng, introducing a new resource type name?
